nix/args.nix

```diff
@@ -4,10 +4,10 @@
     let
       overlays = [
         flakeInputs.rust-overlay.overlays.default
-        (self: super:
-          let toolchain = super.rust-bin.stable.latest; in
-          { cargo = toolchain.minimal; rustc = toolchain.minimal; rustToolchain = toolchain; })
       ];
-    in
-    { pkgs = import flakeInputs.nixpkgs { inherit system overlays; }; };
+    in rec
+    {
+      pkgs = import flakeInputs.nixpkgs { inherit system overlays; };
+      rustToolchain = pkgs.rust-bin.stable.latest;
+    };
```

Comment:
The reason I had to add the overlay was that crane sometimes picked the cargo from nixpkgs even when passed the rust-overlay one in `mkLib`, and of course that one didn't have the wasm32-unknown-unknown target, so builds didn't pass. If CI is green with this change, it's been fixed upstream and we can simplify, that's great.

Reply:
Should be fixed upstream, `prisma-schema-wasm` builds successfully at least.